pkg/ebpf/c/tracer-ebpf.c

```diff
@@ -129,6 +129,53 @@ struct bpf_map_def SEC("maps/port_bindings") port_bindings = {
     .namespace = "",
 };
 
+/* This behaves the same as port_bindings, except it tracks UDP ports.
+ * Key: a port
+ * Value: one of PORT_CLOSED, and PORT_OPEN
+ */
+struct bpf_map_def SEC("maps/udp_port_bindings") udp_port_bindings = {
+    .type = BPF_MAP_TYPE_HASH,
+    .key_size = sizeof(__u16),
```

Comment:
Shouldn't this also incorporate device ID or something like that? what happens when a node has multiple network interfaces?